app/src/main/java/org/dhis2/usescases/searchTrackEntity/SearchTEActivity.java

```diff
@@ -941,9 +946,11 @@ public void setMap(List<SearchTeiModel> teis, HashMap<String, FeatureCollection>
             allItems.addAll(new MapRelationshipToRelationshipMapModel().mapList(searchTeiModel.getRelationships()));
         }
 
+        updateCarousel(allItems);
+
```

Comment:
@mmmateos this needs to go inside the callback otherwise we would have problems initializing...
@Balcan